### ISSUE TITLE
about the detection_result on SportMOT.

### ISSUE THREAD

**Issue by USTBZcb** [Kroery/DiffMOT#4]
Thank you for your excellent work. 
I would like to ask, in the detection results you provided for the SportMOT dataset, only the validation set and test set detection results are available. Would it be possible for you to provide the detection results for the training set as well?

**Comment by GuHuangAI** [comment]
Thanks for your interest.
We do not provide the detection results of the training set since we directly use the gt box of the training set to train our model.
We suggest that you can download the provided [weight of detector](https://github.com/Kroery/DiffMOT/releases/tag/v1.0) to obtain the detection results.
